crates/astria-conductor/src/data_availability.rs

```diff
@@ -446,14 +444,19 @@ async fn fetch_verify_rollup_blob_and_forward_to_assembly(
         "received rollups; verifying"
     );
     rollups.retain(|rollup| {
-        block_verifier::validate_rollup_data(rollup, data.action_tree_root).is_ok()
+        if let Err(e) = rollup.belongs_to(&data) {
+            debug!(reason = &e as &dyn std::error::Error, "dropping rollup");
```

Comment:
log the rollup name?

Reply:
Done, logging `ChainId` as a hex encoded string.